addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -1181,6 +1181,137 @@ QUnit.test('Post a message containing an email address followed by a mention on
     );
 });
 
+QUnit.test('show empty placeholder when no messages', async function (assert) {
```

Comment:
"...when thread contains no messages"

I'm surprised we didn't have a test covering this before?

Reply:
I was too, I searched multiple times but in fact it's "tested" by other tests as they need to check the message list is empty but there were no explicit tests for that